catalog/ui/search-ui/standard/src/main/webapp/templates/search/searchForm.handlebars

```diff
@@ -208,7 +208,7 @@
 
                         <div class="input-group input-group-sm">
                             <span class="input-group-addon">Radius&nbsp;</span>
```

Comment:
It probably could, but removing the name attribute seems like a pretty weak fix.  Will it still cause a refresh problem if I had used a different name?  What's going to prevent the next person from adding the name attribute if they need it, or just want to be consistent with the rest of the form?